python/cuml/dask/cluster/kmeans.py

```diff
@@ -227,3 +238,27 @@ def score(self, X):
             workers=[w]).result() for w, f in gpu_futures.items()]
 
         return np.sum(scores)
+
+    def get_params(self):
+        """
+        Scikit-learn style return parameter state
+        """
+        return self.kwargs
```

Comment:
The sklearn interface supports a `deep=True` deep copy option for some reason.

Reply:
Interesting. I can add that in here.